index.html

```diff
@@ -194,49 +198,228 @@ <h2>
           </dt>
           <dd>
+            <p>Prompts the user for permission to live-capture their display.</p>
             <p>
-              This method operates identically to <code><a>getUserMedia</a></code>, except that it
-              acquires media from display devices, and the <code>constraints</code> argument does
-              not accept <code><a href=
-              "https://w3c.github.io/mediacapture-main/#idl-def-mediatrackconstraints">MediaTrackConstraints</a></code>
-              values. This prevents an application from influencing the selection of sources, see
-              <a href="#constraints"></a> for details.
+              This method is similar to <code><a>getUserMedia</a></code>, except that it
+              acquires media from one display device chosen by the end-user each time.
+              The user agent MUST let the end-user choose which display surface to share
+              out of all available choices every time, and MUST NOT use
+              <code>constraints</code> to limit that choice. Instead,
+              <code>constraints</code> MUST be applied to the media chosen by the user,
+              only after they have made their selection. This prevents an application
+              from influencing the selection of sources, see <a href="#constraints"></a>
+              for details.
             </p>
             <p>
               In addition to drawing from a different set of sources and requiring user selection,
               <code><a>getDisplayMedia</a></code> also differs from
-              <code><a>getUserMedia</a></code> in that permissions cannot be persisted.
+              <code><a>getUserMedia</a></code> in that "granted" permissions cannot be persisted.
             </p>
+            <p>When the <code><dfn>getDisplayMedia()</dfn></code>
+            method is called, the User Agent MUST run the following
+            steps:</p>
+            <ol>
+              <li>
+                <p>Let <var>constraints</var> be the method's first
+                argument.</p>
+              </li>
+              <li>
+                <p>For each member
+                <a href="https://heycam.github.io/webidl/#dfn-present">present</a>
+                in <var>constraints</var> whose value, <var>value</var>, is a
+                dictionary, run the following steps:</p>
+                <ol>
+                  <li>
+                    <p>If <var>value</var> contains a member named <code>advanced</code>,
+                    return a promise <a>rejected</a> with a newly
+                    <a data-link-for="exception" data-lt="create">created</a>
+                    <code>TypeError</code>.</p>
+                  </li>
+                  <li>
+                    <p>If <var>value</var> contains a member which in turn
+                    is a dictionary containing a member named either
+                    <code>min</code> or <code>exact</code>, return a promise
```

Comment:
(If it turns out that there are meaningful differences between exact and ideal in the case of boolean constraints we can revisit this separately.)